spec/features/characters/new_spec.rb

```diff
@@ -50,7 +69,62 @@
     end
   end
 
-  scenario "Creating character with icon, description and extant template"
+  scenario "Creating character with icon, description and extant template", js: true do
+    user = login
+    icon = create(:icon, user: user, keyword: 'Example icon')
```

Comment:
Like in the previous test, can we create another couple of random icons here to be sure it selects the correct one?